cs/giving-feedback.md

```diff
@@ -0,0 +1,88 @@
+# Příručka pro dávání zpětné vazby
+
+*Cílem této příručky je seznámit tě se zpětnou vazbou jako s nástrojem, který ti pomůže efektivně rozvíjet sám sebe i lidi kolem.*
+
+---
+
+💡 Jsi teamleader? Zařaď dávání zpětné vazby do své daily routine! Zlepší to vztahy s tvým týmem, ujasníte si očekávání, posílíš to, co podle tebe funguje a zasadíš se o kulturu dávání zpětné vazby v Appliftingu. Podrobnější postup najdeš už brzy v [**Kodexu team leadera**](./team-leader-codex.md).
+
+---
+
+## Co je (a není) zpětná vazba
+
+Všichni pojem “zpětná vazba” známe, ale co v kontextu naší kultury znamená? V naší DNA je touha rozvíjet se a zlepšovat nejen naše okolí, ale i sami sebe. Zpětná vazba je pro nás vodítkem k tomuto zlepšení, proto podporujeme její sdílení napříč firmou. Otevřená komunikace a bezpečné prostředí, ve kterém se nebojíme si sdělovat své názory, jsou u nás uznávanou hodnotou a každodenní jednání každého z nás směřuje k její kultivaci. Proto si o zpětnou vazbu aktivně žádáme a zároveň ji ostatním ochotně poskytujeme.
+	
+Zpětná vazba:
+
+* je způsob rozvoje sebe i druhých;
+* je cesta z naší vlastní nespokojenosti/nekomfortu;
+* je možnost, jak ovlivnit naše okolí;
+* je sdělení, jak vnímáme chování druhých;
+* je založená na upřímnosti, důvěře a snaze porozumět si;
+* je součástí agilního přístupu k práci;
+* zlepšuje týmovou spolupráci;
+* pomáhá objasnit realitu a eliminuje (chybné) domýšlení;
+* není negativní hnidopišství nebo kritika;
+* není hejt, kterým snižujeme něčí osobu či práci.
+
+# Když zpětnou vazbu dávám
+
+V našem pojetí je zpětná vazba metodou rozvoje, proto by měla primárně pomoci tomu, kdo ji přijímá (ne tomu, kdo ji dává). Je zodpovědností poskytovatele zpětné vazby, dát ji tak, aby příjemci byla zpráva srozumitelná. To klade nárok na schopnost pojmenovávat chování a pocity. Konfrontaci se nevyhýbáme a projevujeme v ní zájem o druhého - cílem je druhému pomoci, ne ho zkritizovat.
+
+Pozor na ničivou empatii! Někdy se stane, že stojíme před rozhodnutím, zda dát zpětnou vazbu či nikoli a nakonec se rozhodneme to neudělat, abychom druhému neublížili. Takovému jednání se říká **ničivá empatie** a jedná se o strach z dávání zpětné vazby, abychom neublížili ostatním. Ve skutečnosti nejde o tendenci chránit druhé, ale chránit nás samotné a nevystavovat se nepříjemné konfrontaci. Výsledem je však nečinné přihlížení situaci, kterou bychom svým jednáním mohli zlepšit.
+
+## Jak zpětnou vazbu formulovat - obecné zásady
+
+**Cíl:** Chceme prohloubit poznání hodnoceného člověka, jeho silných i slabých stránek a osobnostního rozvoje, což tvoří základ pro další zvyšování pracovního výkonu
+
+**Jasnost, srozumitelnost, konkrétnost:** mysli na to, že zpětná vazba je zkreslená šumem způsobeným interpersonální komunikací (např. tón hlasu, neverbální komunikace, atribuční chyby atd.). Tvá komunikace musí být i přesto dostatečně srozumitelná a jasná, nikoli vágní. K tomu ti pomůže uvádět konkrétní příklady.
```

Comment:
Tady pro mě není srozumitelné, co jsou to atribuční chyby, tak by to chtělo buď vysvětlit nebo vynechat...